docker/Dockerfile

```diff
@@ -0,0 +1,70 @@
+#
+# Docker layer for building Gramine
+# from https://github.com/gramineproject/gramine/blob/master/packaging/docker/Dockerfile
+#
+
+ARG UBUNTU_IMAGE=ubuntu:22.04
+
+FROM ${UBUNTU_IMAGE}
+
+ARG UBUNTU_CODENAME=jammy
+WORKDIR /usr/src/raiko
+
+ENV IP_NUMBER=0.0.0.0
+ENV PORT_NUMBER=9090
+ENV SGX_INSTANCE_ID=123
+ENV LOG_PATH=/data/log/sgx
```

Comment:
We can add a config file instead of environment and arguements

Reply:
can use .env to share the vars between compose & dockerfile. I think at least the ports need export in compose.yml

Reply:
We need to frequently modify configurations and the number of configuration items will continue to increase.
So, it's not a good idea to place our configurations in env.